fhir-smart/src/test/java/com/ibm/fhir/smart/test/AuthzPolicyEnforcementTest.java

```diff
@@ -67,8 +68,10 @@
 
 public class AuthzPolicyEnforcementTest {
     private static final String PATIENT_ID =     "11111111-1111-1111-1111-111111111111";
-    private static final String OBSERVATION_ID = "11111111-1111-1111-1111-111111111111";
-    private static final String CONDITION_ID =   "11111111-1111-1111-1111-111111111111";
+    private static final String OBSERVATION_ID = "11111111-1111-1111-1111-111111111112";
```

Comment:
not sure why i chose to use a GUID-like value here, but I do think having a couple different resource types with the same id is an interesting test to ensure we're not using the id alone for any access decisions

Reply:
Updated.